lib/hammer_cli_foreman_tasks/task.rb

```diff
@@ -21,21 +34,46 @@ def execute
     end
 
     class ListCommand < HammerCLIForeman::ListCommand
+      extend WithoutNameOption
+      extend ActionField
+
       output do
         field :id, _('ID')
         field :action, _('Name')
-        field :username, _('Owner')
+        field :state, _('State')
+        field :result, _('Result')
         field :started_at, _('Started at'), Fields::Date
         field :ended_at, _('Ended at'), Fields::Date
+        field :username, _('Owner')
+
+        from :humanized do
+          field :errors, _('Task errors'), Fields::List, :hide_blank => true
+        end
+      end
+
+      build_options
+    end
+
+    class InfoCommand < HammerCLIForeman::InfoCommand
+      extend WithoutNameOption
+      extend ActionField
```

Comment:
You need to `include` this module, not `extend`, otherwise it would not work.

Reply:
It works for me both with extend and include, I'm not really sure why

Reply:
Maybe some plugin that you have and I don't or vice-verse? Anyway, I `extend` doesn't work for me at all for this method. It works for `WIthoutNameOption`